src/config.template.cjs

```diff
@@ -222,13 +222,13 @@ module.exports = {
 	 * "2w"
 	 * "3 days"
 	 */
-	excludeRecentSearch: undefined,
+	excludeRecentSearch: "1 year",
```

Comment:
I think my approach to searching with cross-seed might be way different than yours. Perhaps this is something better discussed in a chat, because it seems like a potentially involved conversation.

Reply:
what's your proposal?

Reply:
I'm open to an "only search once" type thing, or some other kind of `excludeOlder` type thing.